index.html

```diff
@@ -70,6 +70,163 @@ <h1>
+              </ul>
+            </div>
+            <button class="green-button">See project</button>
+          </div>
+        </div>
+        <div class="project-card">
+          <div class="project-card_image">
+            
+          </div>
+          <div class="project-card_details">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="stacks">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="green-button">See project</button>
+          </div>
+        </div>
+        <div class="project-card">
+          <div class="project-card_image">
+            
+          </div>
+          <div class="project-card_details">
+            <h3>Multi-Post Stories<br> Gain+Glory</h3>
+            <div class="stacks">
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+            <button class="green-button">See project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <div class="about-me">
+      <h2>About Me</h2>
+      <img src="./images/Mask Group.png" class="background-top-right" alt="">
+      <img src="./images/Frame.svg" class="background-bottom-left" alt="">
+      <p class="content-paragraph"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      <button class="green-button about-button">Get my resume</button>
+      <div class="skills">
+        <ul class="skills-list">
+          <li>
+            <div class="skill-list">
+              <img src="./images/icon-languages.svg" alt="">
+              <h3>Languages</h3>
+              <ul>
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+            </div>
+          </li>
+          <li>
+            <div class="skill-list">
+              <img src="./images/icon-frameworks.svg" alt="">
+              <h3>Frameworks</h3>
+              <ul>
+                <li>React.js</li>
+                <li>Ruby on Rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+                <li>Selenium</li>
+              </ul>
+            </div>
+          </li>
+          <li>
+            <div class="skill-list">
+              <img src="./images/icon-skills.svg" alt="">
+              <h3>Skills</h3>
+              <ul>
+                <li>Database Management</li>
+                <li>Version Control</li>
+                <li>CLI</li>
+                <li>Web Development</li>
+                <li>API Design</li>
+              </ul>
```

Comment:
> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- [ ] As stated in the requirement above about using the exact margins and spacing, I think your `language, frameworks and skill` needs a little left and right margin. 
- - Looking at the screenshot below, you can see that there is a margin left and eight in the Figma design, the lists are not touching the borders, I'll suggest you update this in your design:
- - FIGMA DESIGN:
![Selection_053](https://github.com/NewtonOmbese/Newton-Portfolio/assets/107632780/761a00e4-ed85-41cd-955d-4065e2497135)
- - YOUR DESIGN:
![Screenshot from 2023-06-02 11-12-58](https://github.com/NewtonOmbese/Newton-Portfolio/assets/107632780/2de3b770-354d-4e3f-8b45-8d2ab74f4f55)

Reply:
@V-Blaze I have a query here is how my layout is
![Screenshot from 2023-06-02 14-03-20](https://github.com/NewtonOmbese/Newton-Portfolio/assets/79839603/f80ed32a-95e8-46c1-a639-2c3f558e048a)
![Screenshot from 2023-06-02 14-20-45](https://github.com/NewtonOmbese/Newton-Portfolio/assets/79839603/ea381b3a-c629-4797-9c64-7ea096b6eae3)

But when I try to add more margin to the left and right this is how it looks like more margins this is how it looks again
![Screenshot from 2023-06-02 14-03-06](https://github.com/NewtonOmbese/Newton-Portfolio/assets/79839603/30ea16bf-521d-4aa1-889a-5ba45367c27e)